test/pottery/scan_test.clj

```diff
@@ -58,7 +58,7 @@
                                  [:div (tr "Some text %1" some-arg)]))}))))
 
 (deftest scan-files-test
-  (is (= '(#:pottery.scan{:filename "test-resources/foo.cljc",
-                          :expressions (#:pottery.scan{:value "Some text %1 %2 %3"})})
+  (is (= '({:pottery.scan/filename "test-resources/foo.cljc"
+            :pottery.scan/expressions ({:pottery.scan/value "Some text %1 %2 %3"})})
```

Comment:
@borkdude Could you make it `::sut/` like the others in the namespace? Not saying it's the best alias but for consistency.

Reply:
Yeah, what I hate about this sut stuff is that copy and pasting results from the REPL to tests doesn't work, but I'll just go ahead and change it